### DISCUSSION TITLE
折角 Advent Calendar に初投稿したのに古新聞扱い

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#372] [要望・アイデア・不具合]
下記記事は、12/05に限定共有記事として投稿し、Advent Calendar で12/08に公開されるように登録しました。

【MATLABユーザー定義関数】自由度が高い「subplotの変種」
https://qiita.com/tsubolabo/items/5b6149b7c7d6f71391bc

　しかし、公開結果を見ると、「投稿日」が、限定共有記事として投稿した12/05になっており、「公開日」の12/08は「更新日」としてしか扱われていません。

　これでは、最新記事を見逃さないよう、私のように、「cretated:>一昨日」でフォローしている人には全く目に触れず、スルーされてしまいます。公開に当たって、「投稿日」は「公開日」に付け替えるべきではないでしようか？

**Comment by tanaka-asami415** [comment]
@tsubolabo 
Advent Calendar へご参加いただきありがとうございます！

現在、投稿日は限定共有・全体公開に関わらず、Qiitaに投稿した日となる仕様となっています🙇
投稿日を公開日に更新したいというご要望は他にもいただいていまして　 https://github.com/increments/qiita-discussions/discussions/25
現在進めているプロジェクトの進捗ともあわせて検討させていただきます！

**Comment by tanaka-asami415** [comment]
@tsubolabo 
本件、対応完了のご報告ができておらず申し訳ありません。

カレンダー記事に限らず、「限定共有記事を全体公開した際に、全体公開日を投稿日とする」仕様について、2023年に対応が完了しておりましたが、ご案内ができておりませんでした🙇
